### PR TITLE
XS✔ ◾ New rule: brainstorming-idea-champion

### PR DESCRIPTION
**Pull request by sethdaily**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ Conversation with Adam about brainstorming ideas

> 2. What was changed?

✏️ new rule

> 3. Did you do pair or mob programming (list names)?

✏️ w/ piers

### CODE REVIEW

**Review by camillars — APPROVED** [review]

File: rules/brainstorming-idea-champion/rule.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+During a Brainstorming day, many great ideas are generated, discussed, and refined. But what happens to these ideas once the day is over? Without a dedicated individual to drive an idea forward, it risks being forgotten or losing momentum. This is where the Idea Champion comes in.
+
+<!--endintro-->
+
+The Idea Champion should be someone who believes in the idea's potential.
+
+## The Idea Champion's responsibilities
+
+* **Take accountability** for the idea from the end of the brainstorming day until a Scrum Team has been assigned and development has started
```
Comment (bradystroud) [inline rules/brainstorming-idea-champion/rule.md:22]: Ideas don't always need to turn into Scrum teams - sometimes we decide they are not actioned at all which is a valid outcome.

```suggestion
* **Take accountability** for the idea from the end of the brainstorming day until a Scrum Team has been assigned and development has started or the idea is parked
```

File: rules/brainstorming-idea-champion/rule.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* **Discuss** the idea and answer people's questions
+* **Communicate** the status of the idea in its [GitHub Discussion](/brainstorming-idea-farming)
+* **Push the idea forwards** and get closure (approval/rejection)
+
+### Where it begins
+
+* An Idea Champion should be assigned to each idea for a specific Brainstorming Day. This should be done when [allocating teams](/brainstorming-team-allocation)
```
Comment (bradystroud) [inline rules/brainstorming-idea-champion/rule.md:29]: Will the idea champion replace the 'Team Lead' role mentioned in https://ssw.com.au/rules/brainstorming-team-allocation/#team-allocation
(the guy in change of the brainstorming team)

If yes we should update that rule as well (update term and link to this one)

File: rules/brainstorming-idea-champion/rule.md (hunk 1 above)
Comment (bradystroud) [reply rules/brainstorming-idea-champion/rule.md:22]: I added this comment before reading the last line of the rule, but i still think its valid

File: rules/brainstorming-idea-champion/rule.md (hunk 2 above)
Comment (sethdaily) [reply rules/brainstorming-idea-champion/rule.md:29]: No it will not replace

**Pull request merged**